lib/graphiti/scoping/filter.rb

```diff
@@ -53,6 +53,7 @@ def each_filter
           type = Types[filter.values[0][:type]]
           unless type[:canonical_name] == :hash || !value.is_a?(String)
```

Comment:
Seems like this would not apply to hash types - would it be possible to move out of this block so all types could support `null`?